src/platforms/hosted/bmp_libusb.c

```diff
@@ -28,10 +28,10 @@
 
 void bmp_ident(bmp_info_t *info)
 {
-	DEBUG_INFO("BMP hosted %s\n for ST-Link V2/3, CMSIS_DAP, JLINK and "
+	PRINT_INFO("Black Magic Debug App %s\n for ST-Link V2/3, CMSIS_DAP, JLINK and "
```

Comment:
I think this list should also include Black Magic Probe in that list. Currently it makes it seem that it only supports 3rd party probes.

Reply:
Ah, good catch!